web/modules/custom/dept_landing_pages/dept_landing_pages.info.yml

```diff
@@ -0,0 +1,8 @@
+name: 'Dept Landing Pages'
```

Comment:
I'd rename this to 'Departmental sites: Landing pages' to keep it consistent with the other modules